ospo/settings.yml

```diff
@@ -50,7 +50,10 @@ url: https://ospo.landscape2.io
 categories:
   - name: OSPO Adopter
     subcategories:
-      - OSPO Adоpter
+      - Enterprise
+      - Public Sector
+      - Nonprofit
+      - Academic
```

Comment:
related to my comment above. I can first go to `landscape.yml` and make the changes before merging this PR 🙂

Reply:
Sounds great!

Let's do something first to make sure this set of changes works fine. As you've seen in the settings file, we've overriding the `OSPO Adopter` category to specify the order in which we display its subcategories. The problem with this approach is that *only* the subcategories listed here will be displayed (others will be ignored). So when you add some more, they won't show up as it'll be annoying.

My suggestion here is to omit the categories section temporarily. This way, you can reorganize everything as you prefer in the `landscape.yml` file and everything will work fine. The only difference will be that the order in which the subcategories will be displayed may vary depending on the window size.

Once you are ready, you can open a PR to setup the groups using the new categories. In the same PR, you could add back the `OSPO Adopter` category to specify the order of the subcategories if you'd like (if this is important for you, of course, otherwise it's better to leave it dynamic).

How does this sound? 🙂

Reply:
@tegioz I've created the PR in the `landscape.yml` could you please confirm the structure is aligned with the changes requested in this `settings.yml` file?

https://github.com/todogroup/ospolandscape/pull/214/files

Reply:
Well if you have it already we may not need this 😇

Let me take a look 👍

Reply:
Can you take a look at the validation result? It looks like there is some sort of indentation problem:

https://github.com/todogroup/ospolandscape/actions/runs/8509469370/job/23305055994?pr=214

I would strongly recommend to make that check required before a PR can be merged, to avoid breaking the landscape.

Reply:
In addition to that, there isn't yet a category named `OSPO Associate`. This affects the groups defined below.

Reply:
Let me know if you need any help with this @anajsana 🙂